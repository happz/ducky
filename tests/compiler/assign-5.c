struct foo {
  int i;
  int j;
  char c;
  int k;
  int l;
};

void fn()
{
  struct foo f, *g;

  g = &f;
}
