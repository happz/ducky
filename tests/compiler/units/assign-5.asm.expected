
  .include "defs.asm"
  .section .rodata


.L1_global:


  .text
  .global fn

fn:
  sub sp, 12
  push r12


  mov r12, fp
  add r12, -10

.L2_fn_return:
  pop r12
  add sp, 12
  ret


_start:
  call &main
  int $INT_HALT
  .global _start
